examples/bert/bert_nvmo_ptq.json

```diff
@@ -0,0 +1,39 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "model_loader": "load_pytorch_origin_model",
+            "model_script": "user_script.py",
+            "io_config": {
+                "input_names": ["input_ids", "attention_mask", "token_type_ids"],
+                "input_shapes": [[1, 128], [1, 128], [1, 128]],
+                "input_types": ["int64", "int64", "int64"],
+                "output_names": ["output"]
+            }
+        }
+    },
+    "passes": {
+        "conversion": {
+            "type": "OnnxConversion",
+            "config": {
+                "target_opset": 17
+            }
+        },
+        "transformers_optimization": {
+            "type": "OrtTransformersOptimization",
+            "disable_search": true,
+            "config":{
+                "model_type": "bert"
```

Comment:
Hi, I ran the example locally with the newly released linux package and it worked. 
![image](https://github.com/microsoft/Olive/assets/94929125/ed4e585e-6ee6-4dc2-a587-24c9d877be22)

Can you add `"opt_level":0` here? Otherwise, the transformers optimizer tries to use an inference session with the ep to do further session graph fusions. but that doesn't work if the ep is tensorrt as the session graph cannot be saved.